packages/design-system/src/index.ts

```diff
@@ -1,42 +1,41 @@
 import './tailwind.css'
 
-export * from './Button'
-export * from './Checkbox'
-export * from './Collapsible'
-export * from './ColorPicker'
-export * from './Countdown'
-export * from './CycleCountdown'
-export * from './CycleProgress'
-export * from './DateChanger'
-export * from './Dropdown'
-export * from './Header'
-export * from './Modal'
-export * from './Navigation'
-export * from './NotificationBadgeWrapper'
-export * from './Progress'
-export * from './Prose'
-export * from './Select'
-export * from './Slider'
-export * from './StepProgress'
-export * from './Switch'
-export * from './Table'
-export * from './Tabs'
+export { default as Button } from './Button'
```

Comment:
If we only export the `default` under the corresponding name here, will the corresponding exported interface props still be available (not tested)? In some cases, e.g. for the select component, the corresponding types have probably been used in KlickerUZH